main/tests/server/src/com/google/refine/operations/column/ColumnAdditionByFetchingURLsOperationTests.java

```diff
@@ -101,9 +101,8 @@ public class ColumnAdditionByFetchingURLsOperationTests extends RefineTest {
             "    \"status\" : \"pending\"\n" +
             " }";
 
-    @Override
     @BeforeTest
-    public void init() {
+    public void initOperation() {
```

Comment:
This seems unrelated

Reply:
If I remember correctly, it's a necessary change for the tests to pass after adding the GREL registration to `RefineTest` instead of being done statically (otherwise, this `init` method overrides that of `RefineTest` which doesn't get executed for this test).